xbmc/windowing/WinSystem.h

```diff
@@ -217,6 +217,11 @@ class CWinSystemBase
 
   virtual std::vector<std::string> GetConnectedOutputs() { return {}; }
 
+  // Settings helper methods
+  bool IsHDRDisplaySettingEnabled();
```

Comment:
I understand that the name comes from the setting string, but the setting string is confusing and so is this function name. Something more like "IsAdjustHDRSettingEnabled" / "IsAutoSwitchHDRSettingEnabled" maybe?
Or keep the name, but add a doxy.

Reply:
added doxygen

Reply:
The method has to use the same name as the setting it references. If the name of the setting is not appropriate (or could be improved) it can be changed to another PR.

Reply:
OK thanks for the doxy, the [code guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#92-doxygen) ask for a different syntax (\brief instead of @brief)